actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -223,6 +223,12 @@ class Parameters
     #    config.always_permitted_parameters = %w( controller action format )
     cattr_accessor :always_permitted_parameters, default: %w( controller action )
 
+    class << self
+      def nested_attribute?(k, v)
```

Comment:
Since this is a separate method now I think we can name these parameters `key` and `value`.

Reply:
Also this should have `# :nodoc:` too since it is public.